kolibri_explore_plugin/assets/src/components/AboutFooter.vue

```diff
@@ -95,6 +101,45 @@
         return plugin_data.androidApplicationId != '' && plugin_data.windowsApplicationId != '';
       },
     },
+    created() {
```

Comment:
For better component lifecycle let's move the `addEventListener` to the `mounted` stage, and the `removeEventListener` to the `beforeDestroy` stage. See https://v2.vuejs.org/v2/guide/instance.html#Lifecycle-Diagram